### PR TITLE
Middleware/1.3.19

### PR DESCRIPTION
**Pull request by TSchmiedlechner**

### CODE REVIEW

File: middleware/middleware-1.3.19.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+## Feature: Switch connected TSE without creating a new Queue
+In the last weeks, we've already received lots of feedback and questions from partners who needed to switch TSEs due to numerous reasons. Until now, this was only possible indirectly by also creating a new Queue and retiring the old one. However, this leads to additional effort on all sides, and will be even more inconvenient when cloud TSEs that are currently in certification will need to be replaced by their certified follow-up version.
+
+We've therefore implemented a way to switch SCUs and therefore TSEs of existing Queues (SCUs are "bound" to a specific TSE and therefore cannot be re-used with other TSEs).
+
+In short, the switch process consists of these steps:
+1. The SCU switch is prepared in the portal, i.e. the target SCU is added to the Cashbox and marked as _switch target_.
+2. The Middleware is restarted to pull the latest changes after the config was rebuilt.
+3. The switch is initiated with an "initiate SCU switch" receipt. After this, no SCU is connected to the Queue anymore.
```
Comment (mijomilicevic) [inline middleware/middleware-1.3.19.md:21]: do you think we should add the daily closing as it is needed to be able to proceed the initiate scu switch?

File: middleware/middleware-1.3.19.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- _fiskaltrust.Middleware.SCU.DE.Epson v1.3.19_
+- _fiskaltrust.Middleware.SCU.DE.Swissbit v1.3.19_
+- _fiskaltrust.Launcher v1.3.19 **_
+
+\* An RC version was released to be able to deliver the bug fixes quickly. After verifying the correct behavior together with our partners, the versions will be re-labeled to non-RC.
+
+** Unlike regular package updates, this change requires a re-download of the Launcher from the Portal. The fiskaltrust.Launcher is currently not able to update itself.
+
+## Next steps in the Middleware
+We will continue to improve the stability of our Middleware in the next sprints. As always, we're happy to hear feedback and suggestions via [info@fiskaltrust.at](mailto:info@fiskaltrust.at) or directly via issues in our GitHub repositories.
```
Comment (mijomilicevic) [inline middleware/middleware-1.3.19.md:93]: should be ".de" i guess

Comment (TSchmiedlechner) [reply middleware/middleware-1.3.19.md:93]: We've always had .at here, as the same goes for the Portal release notes and technically both are wrong 😄 
I hope that we'll have a general email address for this stuff soon.

File: middleware/middleware-1.3.19.md (hunk 1 above)
Comment (TSchmiedlechner) [reply middleware/middleware-1.3.19.md]: Good one, thanks! Added that 👍

**Review by mijomilicevic — APPROVED** [review]

**Pull request merged**